lobby/src/main/java/org/triplea/lobby/server/db/UserController.java

```diff
@@ -89,7 +89,7 @@ public void updateUser(final DBUser user, final HashedPassword hashedPassword) {
 
   /**
    * Workaround utility method.
-   * Should be removed in the next incompatible release.
+   * Should be removed in the next lobby-incompatible release.
```

Comment:
side-note, we're not consistent here about labeling this with `TODO`. IMO ideally we'd have a github issue to track these items and would have a common token that we could search for in the code to find all these examples.

Reply:
Agreed. Labelling is an issue. The token 'incompatible' is pretty good to search for those issues, but it's hard to see what exactly is being affected.
For example: The Delegate I modified in this PR does also extend IRemote. So changing this delegate does also make the client incompatible via network, but doesn't affect the lobby.
So at some point we need to talk about terminology.
In an ideal world we wouldn't even serialize stuff and send it over the network. It would work for a web-app where you control the version the user uses, and can change backed and frontend simultaneously, but not with TripleA.